R/utils-read.R

```diff
@@ -308,27 +308,34 @@ readApprovalPoints <- function(approvals, points, timezone, legend_nm, appr_var_
   
   date_index_list <- list(list(type="Approved",approved_index), list(type="In Review",review_index), list(type="Working",working_index))
   
-  for(sub_list in date_index_list){
```

Comment:
can we remove the semicolons from L301-303. They don't hurt, but they are also not being used in our repgen code anywhere else